gensim/summarization/textcleaner.py

```diff
@@ -52,11 +133,49 @@ def replace_with_separator(text, separator, regexs):
 
 
 def get_sentences(text):
+    """Sentence generator from provided text. Sentence pattern set in `RE_SENTENCE`.
+
+    Parameters
+    ----------
+    text : str
+        Input text.
+    
+    Yields
+    ------
+    str
+        Single sentence extracted from text.
+
+    Example
+    -------
+    >>> text = "Does this text contains two sentences? Yes, it is."
+    >>> for sentence in get_sentences(text):
+    >>>     print(sentence)
+    Does this text contains two sentences?
+    Yes, it is.
+    """
     for match in RE_SENTENCE.finditer(text):
         yield match.group()
 
 
 def merge_syntactic_units(original_units, filtered_units, tags=None):
+    """Processes given sentences and its filtered (tokenized) copies into 
+    SyntacticUnit type. Also adds tags if they are provided to produced units.
+    Returns a SyntacticUnit list. 
+
+    Parameters
+    ----------
+    original_units : list
+        List of original sintences.
+    filtered_units : list
+        List of tokenized sintences.
+    tags : list
+        List of strings used as tags for each unit. None as deafault.
+    
+    Returns
+    -------
+    list
+        SyntacticUnit for each input item.
```

Comment:
Need to use link to type, like :class:`~gensim.summarization.syntactic_unit.SyntacticUnit`